packages/react-components/react-drawer/stories/src/Drawer/DrawerMultipleLevels.stories.tsx

```diff
@@ -148,43 +135,27 @@ export const MultipleLevels = () => {
         </DrawerHeader>
 
         <div className={styles.body}>
-          {level1Motion.canRender && (
-            <DrawerBody
-              ref={level1Motion.ref}
-              className={mergeClasses(
-                styles.level,
-                motionStyles.level,
-                motionStyles.level1,
-                level1Motion.active && motionStyles.levelVisible,
-              )}
-            >
+          <BodyPresenceMotion level={1} visible={level === 1}>
+            <DrawerBody>
               <DrawerHeaderTitle>Level 1 title</DrawerHeaderTitle>
               <p>Level 1 content</p>
             </DrawerBody>
-          )}
-
-          {level2Motion.canRender && (
-            <DrawerBody
-              ref={level2Motion.ref}
-              className={mergeClasses(
-                styles.level,
-                motionStyles.level,
-                motionStyles.level2,
-                level2Motion.active && motionStyles.levelVisible,
-              )}
-            >
+          </BodyPresenceMotion>
+
+          <BodyPresenceMotion level={2} visible={level === 2}>
```

Comment:
Same here.

Reply:
Done in 2910230aaf826acba6d2a5a0678b53e06983f144